MobileDetect.json

```diff
@@ -302,9 +335,9 @@
                 "\\b(Android.*\\bQ8\\b|Sirius[ _]?EVO[ _]?QS|Sirius[ _]?QS|Spirit[ _]?S)\\b"
             ],
             "ECSTablet": "V07OT2|TM105A|S10OT1|TR10CS1",
-            "StorexTablet": "eZee[_']?(Tab|Go)[0-9]+|TabLC7|Looney Tunes Tab",
+            "StorexTablet": "eZee[_"]?(Tab|Go)[0-9]+|TabLC7|Looney Tunes Tab",
```

Comment:
this looks like a diff bug from Github 😄

Reply:
Isn' t it the single-quote/double-quote difference after "eZee[_ ?